src/core/aws-net-websocket-protocol.ads

```diff
@@ -68,4 +73,17 @@ package AWS.Net.WebSocket.Protocol is
    function End_Of_Message (Protocol : State) return Boolean is abstract;
    --  Returns True if we have read a whole message
 
+   procedure Add_Connect_Headers
+     (Protocol : State;
+      URI      : String;
+      Headers  : in out AWS.Headers.List) is abstract;
+   --  Add all required headers to establish a websocket connection to a server
+
+   function Check_Connect_Response
+     (Protocol_Ignored : State;
```

Comment:
This one is actually required by the compiler, otherwise we get a warning about unused variable. Yes, we have already opened a ticket that in the case of expression functions such warnings are useless (and have to change hundreds of cases in our code :-(